index.html

```diff
@@ -39,33 +44,15 @@ <h2 class="sd">
                 </span>    
                 </section>
 
-<section class="work-section">
+
                  <div class="card-sections">
-                    <h1><br>My Recent <br>Works</h1>
-                    
+                    <h1><br>My Recent Works</h1>
+                   
                     <div class="indicator">
                     <a href="#"><img src="./images/Indicator.png" alt="#"></a>
                 </div>
             </div>
-               
-                    <div class="container-card">
-                    <div class="placeholder">         
-                    </div>
-                    <div class="card-lists">
-                        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
-                        <div class="items">
-                        <ul>
-                            <li>Ruby on rails</li>
-                            <li>Css</li>
-                            <li>Java script</li>
-                            <li>Html</li>
-                        </ul>
-                    </div>
-                    <div class="button-cont">
-                        <button type="button" class="potfolio-but">See Project</button>
```

Comment:
- [ OPTIONAL] Try to improve your design on this section:
- You can use `text-aline: center` to your `card-lists` container,
- And Add `border-radius` to the language `li`



![back](https://user-images.githubusercontent.com/79058364/161383608-8d10b0d0-19c5-4469-803a-1e17bb1fe895.png)